src/features/hotspots/checklist/HotspotChecklist.tsx

```diff
@@ -129,72 +130,93 @@ const HotspotChecklist = ({
     [dataTransferTxn, t],
   )
 
-  const checklistData: ChecklistItem[] = [
-    {
-      key: 'checklist.blocks',
-      title: t('checklist.blocks.title'),
-      description: syncStatus,
-      complete:
-        (blockHeight &&
-          hotspot?.status?.height &&
-          blockHeight - hotspot.status.height < 500) ||
-        false,
-      showAuto: true,
-      background: 1,
-    },
-    {
-      key: 'checklist.status',
-      title: t('checklist.status.title'),
-      description: hotspotStatus,
-      complete: hotspot?.status?.online === 'online',
-      showAuto: false,
-      completeText: t('checklist.online'),
-      background: 1,
-    },
-    {
-      key: 'checklist.challenger',
-      title: t('checklist.challenger.title'),
-      description: challengerStatus,
-      complete: challengerTxn !== undefined,
-      showAuto: true,
-      background: 2,
-    },
-    {
-      key: 'checklist.challenge_witness',
-      title: t('checklist.challenge_witness.title'),
-      description: challengeWitnessStatus,
-      complete: witnessTxn !== undefined,
-      showAuto: true,
-      background: 4,
-    },
-    {
-      key: 'checklist.witness',
-      title: t('checklist.witness.title'),
-      description: witnessStatus,
-      complete: witnesses && witnesses.length > 0,
-      showAuto: true,
-      background: 4,
-    },
-    {
-      key: 'checklist.challengee',
-      title: t('checklist.challengee.title'),
-      description: challengeeStatus,
-      complete: challengeeTxn !== undefined,
-      showAuto: true,
-      autoText: t('checklist.auto_hours'),
-      background: 3,
-    },
-    {
-      key: 'checklist.challengee',
-      title: t('checklist.data_transfer.title'),
-      description: dataTransferStatus,
-      complete: dataTransferTxn !== undefined,
-      showAuto: true,
-      background: 3,
-    },
-  ]
-
-  checklistData.sort((a, b) => Number(b.complete) - Number(a.complete))
+  const checklistData: ChecklistItem[] = useMemo(
+    () => [
+      {
+        key: 'checklist.blocks',
+        title: t('checklist.blocks.title'),
+        description: syncStatus,
+        complete:
+          (blockHeight &&
+            hotspot?.status?.height &&
+            blockHeight - hotspot.status.height < 500) ||
```

Comment:
I'm curious what the significance of 500 is? Maybe we could make it a named constant?

Reply:
We just added some buffer room since they don't always get to 1 block away it prevents false positives. Ill add a constant